src/hb/face.rs

```diff
@@ -329,7 +361,7 @@ impl<'a> hb_font_t<'a> {
 }
 
 #[derive(Clone, Copy, Default)]
-pub struct GlyphExtents {
+pub struct hb_glyph_extents_t {
```

Comment:
I think it breaks public API. We should hide any `hb_` types from the public API. You can override it in `lib.rs` and test via `cargo doc`.

Reply:
From what I can tell it's not exposed... only via the deserializer.

Reply:
I guess we're better of marking every `hb_` type with `pub(crate)` and not just `pub` just in case.